api/v2/openapi.yaml

```diff
@@ -496,6 +557,111 @@ components:
           maxItems: 100
           items:
             $ref: '#/components/schemas/Location'
+    Badge:
```

Comment:
Zu einer Badge gehört doch immer die Anzahl, oder? Dann kann man doch direkt ein Objekt mit `type` und `count` draus machen? Dann braucht man auch nicht diesen sperrigen `BadgeAndCount` Datentyp.

Reply:
das Objekt mit type und count gibt es doch in ähnlicher Form. Heißt jetzt aber wie du es angeregt hast Badges.